src/main/scala/com/github/traviscrawford/spark/dynamodb/BaseScanner.scala

```diff
@@ -52,17 +54,31 @@ private[dynamodb] trait BaseScanner {
       .withTotalSegments(config.totalSegments)
       .withSegment(config.segment)
 
-    config.maybeRequiredColumns
-      .foreach(requiredColumns => scanSpec.withProjectionExpression(requiredColumns.mkString(",")))
+    // Parse any projection expressions passed in
+    val exprSpecBuilder = config.maybeRequiredColumns
+      .map(requiredColumns => new ExpressionSpecBuilder().addProjections(requiredColumns: _*))
```

Comment:
Re-introduce `ExpressionSpecBuilder` which was being used in a previous version.